xbmc/network/win10/NetworkWin10.cpp

```diff
@@ -177,16 +180,12 @@ void CNetworkWin10::CleanInterfaceList()
 std::vector<CNetworkInterface*>& CNetworkWin10::GetInterfaceList(void)
 {
   CSingleLock lock (m_critSection);
```

Comment:
This method isn't really thread safe as the pointers may be deleted after the vector is returned. Would be better to store by value in the vector instead of pointers.

This isn't really anything to do with this pr but something we should fix.

Reply:
Yes I know. It's a root issue. current fix just makes it rare